src/OrchardCore/OrchardCore.ContentManagement.Display/Razor/CssOrchardHelperExtensions.cs

```diff
@@ -1,18 +1,22 @@
 using System.Collections.Generic;
+using Microsoft.AspNetCore.Html;
 using OrchardCore.ContentManagement.Metadata.Models;
 using OrchardCore.Mvc.Utilities;
 
 namespace OrchardCore;
 
 public static class CssOrchardHelperExtensions
 {
-    private const string FieldWrapperPrefix = "field-wrapper";
     private const string PartWrapperPrefix = "content-part-wrapper";
+    private const string FieldWrapperPrefix = "field-wrapper";
 
-    public static string GetPartWrapperCssClasses(this IOrchardHelper helper, ContentTypePartDefinition partDefinition, params string[] additionalClasses)
+    public static IHtmlContent GetPartWrapperClasses(this IOrchardHelper helper, ContentTypePartDefinition partDefinition, params string[] additionalClasses)
     {
+        var options = helper.GetThemeOptions();
+
         var items = new List<string>()
```

Comment:
Do we still have to create an extra list. Can't all fragments be provided directly to the GetHtmlBuilder?

Reply:
No need. I removed it with the updated code.